src/test/kotlin/org/spongepowered/mod/FakeEventUser.java

```diff
@@ -0,0 +1,20 @@
+package org.spongepowered.mod;
+
+import org.spongepowered.api.event.Event;
+import org.spongepowered.api.event.EventListener;
+
+public class FakeEventUser<T extends Event> implements EventListener<T> {
```

Comment:
Why is this java file in a kotlin package?

Reply:
I added this when debugging a strange Intellij issue, which appears to have mysteriously gone away. I'll remove it.